user_space/get_fpga.sh

```diff
@@ -26,7 +26,7 @@ cd $OPENWIFI_DIR/
 git submodule init openwifi-hw
 git submodule update openwifi-hw
 cd openwifi-hw
-git checkout master
+git checkout antsdr
```

Comment:
same as above

Reply:
ooops! This is our develop branch, during our work, we should get the new fpga image from openwifi-hw which support antsdr, so we edit the submodule uri. Once these pull request is received by Jiao, he can change this to the opensdr master branch.